packages/@aws-cdk/aws-rds/README.md

```diff
@@ -45,33 +45,85 @@ By default, the master password will be generated and stored in AWS Secrets Mana
 Your cluster will be empty by default. To add a default database upon construction, specify the
 `defaultDatabaseName` attribute.
 
+### Starting an Instance Database
+To set up a instance database, create an instance of `DatabaseInstance`. You must
```

Comment:
We use the terminology "define a `DatabaseInstance`" (instead of "create an instance").